src/backend/binaryen/utils.ts

```diff
@@ -1623,18 +1622,67 @@ export const enum VtableFieldIndex {
     META_INDEX = 0,
 }
 
-export const enum MetaFieldOffset {
+export const SIZE_OF_META_FIELD = 12;
```

Comment:
What does this mean?

Reply:
> What does this mean?

this means the size of bytes occupied by the meta field, it helps to traverse every meta field.